app/components/visualizer/main-viz/templates/Chart.jsx

```diff
@@ -0,0 +1,42 @@
+import DynamicPieChart from "./DynamicPieChart.js";
+
+class Chart extends React.Component {
+  componentDidMount() {
+    let el = React.findDOMNode(this);
+
+    $.getJSON("blobs/deptGrades.json", (data) => {
+      this.chart = new DynamicPieChart({
+        id: "_id",
+        data: data,
+        element: "#main-viz",
+        height: 512,
```

Comment:
We may want to choose a canvas size that's pretty large and stick with it as a base for awhile.  Then later we can figure out what we want to do about resizing.  The issue is that SVG's can create huge windows and I'm unsure we want that.